internal/mode/static/nginx/config/servers.go

```diff
@@ -41,6 +41,22 @@ var httpBaseHeaders = []http.Header{
 		Name:  "Connection",
 		Value: "$connection_upgrade",
 	},
+	{
+		Name:  "X-Real-IP",
+		Value: "$remote_addr",
+	},
+	{
+		Name:  "X-Forwarded-Proto",
+		Value: "$scheme",
+	},
+	{
+		Name:  "X-Forwarded-Host",
+		Value: "$host",
+	},
+	{
+		Name:  "X-Forwarded-Port",
+		Value: "$server_port",
+	},
```

Comment:
Are these required to be set?

Reply:
Yes these are required to be set to story proxy information, sorry replied on the wrong thread